README.md

```diff
@@ -26,9 +36,9 @@ lint_and_unit: &lint_and_unit
   - lint-yaml
   - lint-markdown
 
-version: 2.1
+version: 2.1  # this is circle version
```

Comment:
As discussed in Slack, these comments are a little bit 😞 
Should we link to the CircleCI documentation instead?